src/app/components/audio-analysis/pages/list/list.component.ts

```diff
@@ -1,23 +1,119 @@
 import { Component } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { AnalysisJobsService } from "@baw-api/analysis/analysis-jobs.service";
+import { Filters } from "@baw-api/baw-api.service";
+import { projectResolvers } from "@baw-api/project/projects.service";
 import {
   audioAnalysesMenuItem,
   audioAnalysisCategory,
-  newAudioAnalysisMenuItem,
+  newAudioAnalysisJobMenuItem,
 } from "@components/audio-analysis/audio-analysis.menus";
-import { PageComponent } from "@helpers/page/pageComponent";
+import { PagedTableTemplate } from "@helpers/tableTemplate/pagedTableTemplate";
+import { Param } from "@interfaces/apiInterfaces";
+import { AnalysisJob } from "@models/AnalysisJob";
+import { Project } from "@models/Project";
 import { List } from "immutable";
+import { DateTime } from "luxon";
+import { Observable } from "rxjs";
+
+interface TableRow {
+  name: Param;
+  scripts: AnalysisJob;
+  creator: AnalysisJob;
+  started: DateTime;
+  status: string;
+  statusUpdated: DateTime;
+  model: AnalysisJob;
+}
+
+const projectKey = "project";
 
-//TODO: OLD-CLIENT REMOVE
 @Component({
   selector: "baw-audio-analyses",
-  template: "<baw-client></baw-client>",
+  templateUrl: "list.component.html",
 })
-class AudioAnalysesComponent extends PageComponent {}
+class AudioAnalysesComponent extends PagedTableTemplate<TableRow, AnalysisJob> {
+  public constructor(
+    protected api: AnalysisJobsService,
+    protected route: ActivatedRoute
+  ) {
+    super(api, (analysisJobs) =>
+      analysisJobs.map((analysisJob) => ({
+        name: analysisJob.name,
+        scripts: analysisJob,
+        creator: analysisJob,
+        started: analysisJob.startedAt,
+        status: analysisJob.overallStatus,
+        statusUpdated: analysisJob.overallStatusModifiedAt,
+        model: analysisJob,
+      }))
+    );
+  }
+
+  public columns = [
+    { name: "Id" },
+    { name: "Name" },
+    { name: "Scripts" },
+    { name: "Creator" },
+    { name: "Started" },
+    { name: "Status" },
+    { name: "Status Updated" },
+    { name: "Actions" },
+  ];
+
+  public sortKeys = {
+    id: "id",
+    name: "name",
+    started: "startedAt",
+    status: "overallStatus",
+    statusUpdated: "overallStatusModifiedAt",
+  };
+
+  public get project(): Project | null {
+    const routeProjectModel = this.route.snapshot.data[projectKey];
+    if (routeProjectModel) {
+      return routeProjectModel.model;
+    }
+
+    return null;
+  }
+
+  protected override apiAction(
+    filters: Filters<AnalysisJob>
+  ): Observable<AnalysisJob[]> {
+    const filterByProject: Filters<AnalysisJob> = {
+      filter: {
+        or: {
+          projectId: { eq: this.project?.id },
```

Comment:
probably shouldn't include this filter condition if the project id is null

Reply:
![image](https://github.com/user-attachments/assets/63aa2beb-2a8b-44da-8d80-93591347c1b9)

Line below handles this condition

However, I agree that it's not very clear and it's not very readable

Reply:
oh i see

i'd wrap this in an if statement. the mixed up order is  hard to read.


```
filter = { filter: {}};
if (this.project) {
   filter.filter = { or: ...}
}
```

Reply:
Good idea, I am thinking something like this using the `filterOr` helper

![image](https://github.com/user-attachments/assets/a68fec6b-6720-460e-b993-3f2554a48e3d)